app/Http/Controllers/Api/V1/Admin/UserStatusController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1\Admin;
+
+use App\Models\User;
+use App\Traits\ApiResponser;
+use App\Services\UserService;
+use OpenApi\Attributes as OA;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\User\ChangeStatus;
+
+class UserStatusController extends Controller
+{
+    use ApiResponser;
+
+    private $userService;
+
+    public function __construct(UserService $userService)
```

Comment:
@dhrumin-7span No need take the object in the function param

Reply:
Removed.